package.json

```
@@ -16,7 +17,7 @@
   },
   "scripts": {
     "build": "turbo run build",
-    "code-review": "turbo run typecheck format:check lint:check",
+    "code-review": "turbo run typecheck format:check lint:check test:coverage",
```

`vitest` is monorepo-aware and should be used without turbo to make a `coverage` report for the whole repository

you can copy the configuration (`turbo.json`, `package.json`) from https://github.com/pagopa/io-messages

```suggestion
    "code-review": "turbo run typecheck format:check lint:check //#test:coverage",
```